docs/schema/block-element-blocks-elementworkflowinfo-elementaction4.md

```diff
@@ -1,5 +1,6 @@
 ---
-title: Block element
+title: Block element represents visual building blocks on the workflow designer surface.
+description: Block element
```

Comment:
This description is very short, let's add something like this: "Learn about the Block element that represents visual building blocks on the workflow designer surface."